Describe which mount options are default for NFS (#3939)
owncloud/docs: modules/admin_manual/pages/installation/deployment_recommendations/nfs.adoc

== NFS Mount Options

> Please see the {man-nfs-ubuntu-url}[Ubuntu man pages] for a detailed description of the NFS mount options. Consider the following options are default for NFS except if excplicit set differently when mounting: `rw`, `suid`, `dev`, `exec`, `auto`, `nouser`, and `async`.

Reviewer: ```suggestion
See the {man-nfs-ubuntu-url}[Ubuntu man pages] for a detailed description of the NFS mount options. The following options are default for NFS except if explicitely set differently when mounting: `rw`, `suid`, `dev`, `exec`, `auto`, `nouser`, and `async`.
```

> Dependent on the NFS version used, consider following mount options:

Reviewer: ```suggestion
Dependening on the NFS version used, consider the following mount options:
```

== NFS Mount Options

> See the {man-nfs-ubuntu-url}[Ubuntu man pages] for a detailed description of the NFS mount options. The following options are default for NFS except if explicitely set differently when mounting: `rw`, `suid`, `dev`, `exec`, `auto`, `nouser`, and `async`.

Reviewer 2: ```suggestion
See the {man-nfs-ubuntu-url}[Ubuntu man pages] for a detailed description of the NFS mount options. The following options are default for NFS except if explicitly set differently when mounting: `rw`, `suid`, `dev`, `exec`, `auto`, `nouser`, and `async`.
```

> Dependening on the NFS version used, consider the following mount options:

Reviewer 2: ```suggestion
Depending on the NFS version used, consider the following mount options:
```
